src/lib/registration/ui/addon_selection_base_dialog.rb

```diff
@@ -71,7 +71,13 @@ def addon_widget_id(addon)
       # @param [Boolean] enable true for filtering beta releases
       def filter_beta_releases(enable)
         self.class.filter_beta = enable
-        @addons = enable ? @all_addons.reject(&:beta_release?) : @all_addons
+        if enable
+          unregistered_addons = @all_addons.reject(&:registered?)
```

Comment:
well, not so nice nice, as it is not unregistered, but not yet registered, so maybe available_addons?